README.md

```diff
@@ -421,7 +421,7 @@ If you're running a Kubernetes variant, the no-proxy list will automatically inc
 Beyond just changing the settings above to affect the `admin` and `control` containers, you can add and remove host containers entirely.
 As long as you define the three fields above -- `source` with a URI, and `enabled` and `superpowered` with true/false -- you can add host containers with an API call or user data.
 
-You can optionally define a `user-data` field with arbitrary base64-encoded data, which will be made available in the container at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME/user-data`.
+You can optionally define a `user-data` field with arbitrary base64-encoded data, which will be made available in the container at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME/user-data` and (since Bottlerocket v1.0.8) `/.bottlerocket/host-containers/current/user-data`.
 (It was inspired by instance user data, but is entirely separate; it can be any data your host container feels like interpreting.)
```

Comment:
Would you please also update line 447 that talk about this persistent storage directory?  Perhaps:

```markdown
In addition, all host containers come with persistent storage that survives reboots and container start/stop cycles.
It's available at `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME` and (since Bottlerocket v1.0.8) `/.bottlerocket/host-containers/current`.
```

Reply:
Should we call `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME` deprecated and encourage `/.bottlerocket/host-containers/current/user-data`?

Reply:
If we add `/.bottlerocket/host-containers/current`, we'd create a collision with any host container called "current".

Reply:
> Should we call `/.bottlerocket/host-containers/$HOST_CONTAINER_NAME` deprecated and encourage `/.bottlerocket/host-containers/current/user-data`?

I think we should definitely consider it, but I'm not sure we should confuse things right away by doing it in this PR.  It'd be useful to have a period of overlap to see if there are any issues, and deprecate later.  (This one is fairly low stakes.)

Reply:
Probably good to call out `current` as a "reserved word/name" in the docs, even though we're mentioning it as the location for user data.  Not everyone will make that connection immediately.

Reply:
Oh - maybe I didn't read far enough.  A conditional mount on `container name != "current` seems fine too